[doc] add a tip about using SDXL refiner with hunyuan-dit and pixart (#8735)
huggingface/diffusers: docs/source/en/using-diffusers/sdxl.md

).to("cuda")
```

<Tip warning={true}>

> You can also use SDXL refiner with a different base model. For example,you can use[Hunyuan-DiT](https://huggingface.co/docs/diffusers/api/pipelines/hunyuandit) or [PixArt-Sigma](https://huggingface.co/docs/diffusers/api/pipelines/pixart_sigma) Text-to-Image pipelines to generate images with better prompt adherence. Once you have generated an image, you can then pass it to SDXL refiner to enhance final generation quality.

Reviewer: ```suggestion
You can use SDXL refiner with a different base model. For example, you can use the [Hunyuan-DiT](../../api/pipelines/hunyuandit) or [PixArt-Sigma](../../api/pipelines/pixart_sigma) pipelines to generate images with better prompt adherence. Once you have generated an image, you can pass it to the SDXL refiner model to enhance final generation quality.
```
